Purchases/Public/RCPurchases.m

```diff
@@ -910,22 +915,29 @@ - (void)handleOfferingsUpdateError:(NSError *)error completion:(RCReceiveOfferin
     CALL_IF_SET_ON_MAIN_THREAD(completion, nil, error);
 }
 
-- (void)receiptData:(void (^ _Nonnull)(NSData * _Nonnull data))completion
-{
+- (void)receiptData:(RCReceiveReceiptDataBlock)completion {
+    [self receiptDataWithForceRefresh:NO completion:completion];
+}
+
+- (void)receiptDataWithForceRefresh:(BOOL)forceRefresh completion:(RCReceiveReceiptDataBlock)completion  {
+    if (forceRefresh) {
+        [self refreshReceipt:completion];
```

Comment:
the log lines were actually neat, sorry I forgot to add them in the suggestion 🤦‍♂️